docs/_pages/releases.md

```diff
@@ -13,6 +13,8 @@ sidebar:
 
 * Adding new overloads to all `GreaterOrEqualTo` and `LessOrEqualTo` assertions, adding the word `Than` - [#1673](https://github.com/fluentassertions/fluentassertions/pull/1673)
 
+* Added `BeSingle` and `HaveCount` for `XElement` - [#1681](https://github.com/fluentassertions/fluentassertions/pull/1673)
```

Comment:
Not up-to-date anymore, the same for the PR title, isn't it?

Reply:
Yes, I already saw it. Still working on the branch. I also have to rerun the api check.

Reply:
solved